airflow/providers/snowflake/operators/snowflake.py

```diff
@@ -548,7 +548,7 @@ def execute_complete(self, context: Context, event: dict[str, str | list[str]] |
                 raise AirflowException(msg)
             elif "status" in event and event["status"] == "success":
                 hook = SnowflakeSqlApiHook(snowflake_conn_id=self.snowflake_conn_id)
-                query_ids = cast(List[str], event["statement_query_ids"])
+                query_ids = cast(list[str], event["statement_query_ids"])
```

Comment:
Can’t do this, will fail on Python 3.8.

Reply:
Yeah, `ruff` is doing that and I am switching it back manually, but I must have missed that one.   Thanks for the catch.